cli/src/main/kotlin/com/malinskiy/marathon/cli/config/ConfigFactory.kt

```diff
@@ -67,7 +67,8 @@ class ConfigFactory(private val mapper: ObjectMapper) {
             testOutputTimeoutMillis = config.testOutputTimeoutMillis,
             debug = config.debug,
             vendorConfiguration = vendorConfiguration as VendorConfiguration,
-            analyticsTracking = config.analyticsTracking
+            analyticsTracking = config.analyticsTracking,
+            pullScreenshotFilterConfiguration = config.pullScreenshotFilterConfiguration
```

Comment:
we need to ignore caching for these tests as we do for strict run tests, check `TestCacheLoader:69`. We immediately emit cache miss for strict run tests. Looks like we need to do the same for screenshot tests

Reply:
True, done in a similar manner to what we do for strict run